test/test_game.py

```diff
@@ -10,102 +10,84 @@
 
     @patch("builtins.print")
     def test_prompt_o_for_second_turn(self, mock_print):
-        message = MockMessage()
-        test_player = HumanPlayer("O")
-        game = Game(message=message)
         total_marks_on_board = 1
-        game.prompt_for_move(total_marks_on_board)
-        mock_print.assert_called_with(message.prompt_for_move(test_player))
+        self.game.prompt_for_move(total_marks_on_board)
+        mock_print.assert_called_with(self.message.prompt_for_move(self.test_player_o))
 
     @patch("builtins.print")
     def test_prompt_x_for_third_turn(self, mock_print):
-        message = MockMessage()
-        test_player = HumanPlayer("X")
-        game = Game(message=message)
         total_marks_on_board = 2
-        game.prompt_for_move(total_marks_on_board)
-        mock_print.assert_called_with(message.prompt_for_move(test_player))
+        self.game.prompt_for_move(total_marks_on_board)
+        mock_print.assert_called_with(self.message.prompt_for_move(self.test_player_x))
 
     @patch("builtins.print")
     def test_handle_draw_displays_game_over_message(self, mock_print):
-        message = MockMessage()
-        game = Game(message=message)
-        game.handle_draw()
-        mock_print.assert_called_with(message.game_over_message())
+        self.game.handle_draw()
+        mock_print.assert_called_with(self.message.game_over_message())
 
     def test_zero_turns_player_x_goes_first(self):
-        game = Game()
         total_marks_on_board = 0
-        current_player = game.get_current_player(total_marks_on_board)
+        current_player = self.game.get_current_player(total_marks_on_board)
         self.assertEqual("X", current_player.mark)
 
     def test_one_turn_player_o_goes_next(self):
-        game = Game()
         total_marks_on_board = 1
-        current_player = game.get_current_player(total_marks_on_board)
+        current_player = self.game.get_current_player(total_marks_on_board)
         self.assertEqual("O", current_player.mark)
 
     def test_two_turn_player_x_goes_next(self):
-        game = Game()
         total_marks_on_board = 2
-        current_player = game.get_current_player(total_marks_on_board)
+        current_player = self.game.get_current_player(total_marks_on_board)
         self.assertEqual("X", current_player.mark)
 
     @patch("builtins.input", side_effect=["3"])
     def test_set_player_one_symbol_returns_symbol_with_input_3(self, mock_input):
-        game = Game()
-        message = MockMessage()
-
         expected_symbol = SymbolOptions().get_symbol("3")
 
-        message = message.choose_symbol_player_one()
+        test_message = self.message.choose_symbol_player_one()
 
-        result = game.set_player_symbol(message)
+        result = self.game.set_player_symbol(test_message)
 
         self.assertEqual(expected_symbol, result)
 
     @patch("builtins.input", side_effect=["2"])
     def test_set_player_two_symbol_returns_symbol_with_input_2(self, mock_input):
```

Comment:
this test has the extra spacing between "arrange -> act -> assert"; while this is my preference, the rest of the tests in this project do not use it. Pick a way, and stick with it throughout.

Reply:
I like it that way better too - I'll change them all to be consistent!